services/src/handlers/ebv.rs

```diff
@@ -567,6 +603,12 @@ impl<C: Context> Task<C::TaskContext> for EvbOverviewTask<C> {
             .map_err(ErrorSource::boxed)
     }
 
+    async fn cleanup_on_error(&self, _ctx: C::TaskContext) -> Result<(), Box<dyn ErrorSource>> {
+        // TODO: call remove method on overview creator
```

Comment:
do you still want to do this?

Reply:
Yes, but this is the next step when I create the clean-up functionality.